scripts/zf_mkdist

```diff
@@ -58,10 +58,10 @@ copy_files_to_tmpdir() {
 
 
 grab_artifacts_from_tmpdir() {
-    tmpdir="${zf_tmpdir}/build/artifacts"
+    artifact_tmpdir="${zf_tmpdir}/build/artifacts"
```

Comment:
Shall we also declare it read-only to prevent further accidental overwrites?
```
declare -r tmpdir=$(mktemp -d)
```

Reply:
Sounds like a good idea!

Reply:
New tests kicked off with this change:
http://xcb-rb-logs.xilinx.com/users/sianj/results/2024/10/04/0_dibench_15-05-14